src/feform.js

```diff
@@ -15,7 +15,10 @@ const findInputs = function(children, valueChange, context={inputIndex: 0, child
     React.Children.forEach(children, (child) => {
         // have to assign a key to avoid warning about dynamic children
         // (https://facebook.github.io/react/docs/multiple-components.html#dynamic-children)
-        const childKey = _.isNil(child.key) ? `feinput-child${context.childIndex++}` : child.key;
+        if (_.isNil(child)) {
+            return;
```

Comment:
Since React accepts `null` as valid Components, I don't think we should exclude them by returning early here. Null Components get added in here (line 35):

`inputs.push(child);`

I think what I'd probably do is:

```
let childKey;

if (child) {
  childKey = _.isNil(child.key) ? `feinput-child${context.childIndex++}` : child.key;
}
```

This should safely allow us to call `child.key`.